### ISSUE TITLE
Exclude button text in the copied string

### ISSUE THREAD

**Issue by fizerkhan** [sachinchoolur/ngclipboard#24]
I have button inside my HTML. I wanted to copy everything except my button text.
Is there any way to do it without changing my HTML layout?

**Comment by kimslava93** [comment]
I don't know if it is still not resolved, but you can use 
```
new Clipboard('.btn', {
    text: function(trigger) {
        return trigger.getAttribute('aria-label');
    }
});
```

**Comment by fizerkhan** [comment]
@kimslava93 Thanks. Is there any angular way of doing it?

**Comment by kimslava93** [comment]
I don't think so. At least I did not see that in the source code, but you can make a pull request any time, since it just using the same library and just add such feature, which could be useful for others.
